### PR TITLE
Removed billing button & premium features notify from the public code

### PR DESCRIPTION
**Pull request by bsekachev**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat-ui/src/components/login-page/login-form.tsx [hunk 1]
```diff
@@ -156,7 +156,11 @@ function LoginFormComponent(props: Props): JSX.Element {
                         </Form.Item>
                     )
                 }
-                { pluginsToRender.map((Component: React.FunctionComponent, index: number) => <Component key={index} />)}
+                {
+                    pluginsToRender.map(({ component: Component }, index) => (
+                        <Component targetProps={props} targetState={credential} key={index} />
```
Comment (bsekachev) [inline cvat-ui/src/components/login-page/login-form.tsx:161]:
```suggestion
                        <Component targetProps={props} targetState={{credential}} key={index} />
```

File: cvat-ui/src/reducers/index.ts [hunk 2]
```diff
@@ -277,10 +277,16 @@ export interface PluginsState {
         [index: string]: CallableFunction;
     },
     components: {
+        header: {
+            userMenu: {
+                items: [],
```
Comment (bsekachev) [inline cvat-ui/src/reducers/index.ts:282]:
```suggestion
                items: PluginComponent[],
```

File: cvat-ui/src/reducers/index.ts [hunk 3]
```diff
@@ -839,7 +848,7 @@ export interface ReviewState {
 
 export interface OrganizationState {
     list: any[];
-    current: any | null;
+    current?: any | null;
```
Comment (klakhov) [inline cvat-ui/src/reducers/index.ts:851]:
```suggestion
    current?: Organization | null;
```

File: cvat-ui/src/components/header/header.tsx [hunk 4]
```diff
@@ -245,154 +246,159 @@ function HeaderContainer(props: Props): JSX.Element {
...
-                className='cvat-menu-item-highlighted'
-                icon={<UpgradeIcon />}
-                key='upgrade'
-                onClick={() => window.open(upgradeLink, '_self')}
+                icon={<ControlOutlined />}
+                key='admin_page'
+                onClick={(): void => {
+                    // false positive
+                    // eslint-disable-next-line
+                    window.open(`${tool.server.host}/admin`, '_blank');
```
Comment (klakhov) [inline cvat-ui/src/components/header/header.tsx:260]: Do we really have eslint error here? I don't see any problems if I remove `// eslint-disable-next-line` and run eslint localy

Comment (bsekachev) [reply cvat-ui/src/components/header/header.tsx:260]: Before it raised the error like ``unsafe open file`` or something like this, maybe eslint rule was fixed, maybe something else. 
Need to check, if it is not necessary anymore, we can remove.

File: cvat-ui/src/components/cvat-app.tsx [hunk 5]
```diff
@@ -437,6 +437,10 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
             .filter(({ data: { shouldBeRendered } }) => shouldBeRendered(this.props, this.state))
             .map(({ component: Component }) => Component());
 
+        const loggedInModals = pluginComponents.loggedInModals
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply cvat-ui/src/components/cvat-app.tsx:440]: I meant modals which should be rendered when user is logged in. Any proposals?

[the remark this answers is not in the document]
Comment (klakhov) [reply cvat-ui/src/components/cvat-app.tsx:440]: ok, lets keep it this way

**Review by klakhov — APPROVED** [review]

**Pull request merged**